test/e2e/common/empty_dir.go

```diff
@@ -151,7 +151,7 @@ var _ = ginkgo.Describe("[sig-storage] EmptyDir volumes", func() {
 		This test is marked LinuxOnly since Windows does not support setting specific file permissions.
 	*/
 	framework.ConformanceIt("volume on default medium should have the correct mode [LinuxOnly] [NodeConformance]", func() {
-		doTestVolumeMode(f, testImageRootUid, v1.StorageMediumDefault)
+		doTestVolumeMode(f, nil, v1.StorageMediumDefault)
```

Comment:
just to make it clear, maybe add comments after nil /*root user*/?

Reply:
Well. I've updated this a bit. I've changed it from a pointer to just a regular ``int64``. Now, it's either ``0`` or ``nonRootUid``, which makes it a bit more clear, and it's also consistent with the other Volume related tests, which treat the root / non-root tests in a similar fashion.

Additionally, I've added a comment to ``testPodWithVolume`` (which is actually called by all the tests) mentioning that if the UID is 0, the Pod will run with with its default user (``root``, or if we'll add Windows support one day, ``ContainerAdministrator``)